addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -82,23 +97,42 @@ export class CallParticipantCard extends Component {
     }
 
     get hasVideo() {
-        return Boolean(this.rtcSession?.videoStream);
+        return Boolean(this.rtcSession?.videoStreams.size);
     }
 
     get isTalking() {
         return Boolean(this.rtcSession && this.rtcSession.isTalking && !this.rtcSession.isMute);
     }
 
+    get isInset() {
+        return Boolean(this.props.setInset);
```

Comment:
I'm having trouble understanding what this is. This getter currently returns `false` if `setInset` is undefined on `this.props` and `true` otherwise. If you really want to test for the existence of the `setInset` function, I'd suggest renaming the getter to something like `hasSetInsetFn`. But if you want to test whether the card is the “insetCard” of the current call, you're most likely not checking the right thing